src/Gui/MainWindow.cpp

```diff
@@ -2019,8 +2021,13 @@ QPixmap MainWindow::splashImage() const
     if (fi.isFile() && fi.exists())
         splash_image.load(fi.filePath(), "PNG");
 
-    // if no image was found try the config
+    // set a random splash path
+    // TODO: is there any way to not hardcode this?
+    int random = rand() % 14; // 14 is splash count
```

Comment:
How can you find out the current existing number of splash images exist and use that as a variable?

Reply:
Start a counter at zero, start trying to access those resources, the first one that fails is the end of the count.